packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -58,6 +58,22 @@ void _animateColumnControllerToItem(FixedExtentScrollController controller, int
   );
```

Comment:
Can you attach a screenshot for the date picker to help review as well?

Reply:
![Simulator Screen Shot - iPhone 11 - 2020-09-10 at 14 55 35](https://user-images.githubusercontent.com/11473691/92691807-c16b9d80-f375-11ea-9dad-d0b875b46bbb.png)

This is what the CupertinoDatePicker looks like in action, as iOS has removed this style, so it can't be compared to native components.

Reply:
You can by picking wheels in xcode. But it's definitely not very obvious.

![Screen Shot 2020-09-10 at 3 58 28 PM](https://user-images.githubusercontent.com/156888/92820149-7ace7100-f37e-11ea-829e-979b456980a7.png)